### PR TITLE
Admin template

### PR DESCRIPTION
**Pull request by theleebriggs**
## Trello
https://trello.com/c/Qb2wOQuX/129-3-change-the-site-to-use-the-admin-template

## Need

The use case for the app is pivoting towards a more internal one - i.e Theme/Taxonomy work, and as such it was felt that the admin template would be more appropriate.

## Before
before

## After
after

### CODE REVIEW

File: app/assets/stylesheets/_pagination.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.pagination {
+  margin: 0;
+}
+.pagination-label {
+  display: block;
+}
```
Comment (pmanrubia) [inline app/assets/stylesheets/_pagination.scss]: Minor: End of line missing

File: app/views/kaminari/_page.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    page:          a page object for "this" page
+    url:           url to this page
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+-%>
+<li class="<%= 'active' if page.current? %>">
+  <%= link_to page, url %>
+</li>
```
Comment (pmanrubia) [inline app/views/kaminari/_page.html.erb]: Minor: end of line

File: app/assets/stylesheets/_pagination.scss (hunk 1 above)
Comment (theleebriggs) [inline app/assets/stylesheets/_pagination.scss]: Addressed this now.

File: app/views/kaminari/_page.html.erb (hunk 2 above)
Comment (theleebriggs) [inline app/views/kaminari/_page.html.erb]: Addressed this now.

**Comment by theleebriggs** [comment]
All changes made as per your request @pmanrubia

**Comment by pmanrubia** [comment]
👍

**Pull request merged**